configure.ac

```diff
@@ -382,25 +382,10 @@ if test "x$enable_werror" = "xyes"; then
   if test "x$CXXFLAG_WERROR" = "x"; then
     AC_MSG_ERROR("enable-werror set but -Werror is not usable")
   fi
-  AX_CHECK_COMPILE_FLAG([-Werror=gnu],[ERROR_CXXFLAGS="$ERROR_CXXFLAGS -Werror=gnu"],,[[$CXXFLAG_WERROR]])
```

Comment:
I see `-Werror` being removed here, but where is it re-added? :)

Reply:
Here: https://github.com/bitcoin/bitcoin/blob/0179abea02576b4e4291d36d14a2915d3d3c477d/configure.ac#L385

Reply:
Heh, thanks! Literally on the same line :) Sorry.